streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -153,24 +153,19 @@ public StateStore getStateStore(final String name) {
         }
         final ProcessorNode previousNode = currentNode();
         try {
-            final List<ProcessorNode<K, V>> children = (List<ProcessorNode<K, V>>) currentNode().children();
```

Comment:
The changes in this class are just a code simplification as discussed with @bbejeck and @guozhangwang -- no change in behavior.